docs/appendices/release-notes/unreleased.rst

```diff
@@ -58,4 +58,5 @@ None
 Fixes
 =====
 
-None
+- Fixed encoding string array type values that contain escaped double
+  quotes. The issue occurred when using the PostgreSQL wire protocol.
```

Comment:
Maybe this could be rephrased to convey the user perspective a bit better. Something like this?

> Fixed an issue that caused an error when a client using the PostgreSQL wire protocol is used to retrieve array values containing a escaped double quotes. For example, the JDBC client failed with ``Can't extract array data from JDBC array``.


Not sure if it's an improvement.

Reply:
👍 tnx, will use this changelog entry